pandas_schema/validation.py

```diff
@@ -214,6 +215,80 @@ def validate(self, series: pd.Series) -> pd.Series:
         return (series >= self.min) & (series < self.max)
 
 
+def convert_type_to_dtype(type_to_convert: type) -> np.dtype:
```

Comment:
I'm fairly sure that `np.dtype(int)` returns `np.int64`, making this function redundant.

Reply:
I checked some different Python versions. On Linux Ubuntu with Python 3.8.5, it returns indeed `np.int64`. In my IDE on Windows it returns `np.int32`. The latter is for both Python 3.8 and 3.9. These [stackoverflow](https://stackoverflow.com/questions/36278590/numpy-array-dtype-is-coming-as-int32-by-default-in-a-windows-10-64-bit-machine) answers explain that this results from C in Windows, where long int is 32bit despite the system being 64bit. 

So `pd.Series([1,2,3]).dtype` results in `np.int64` and `pd.Series(np.array([1,2,3])).dtype` results in `np.int32`. 

It makes it tricky to anticipate which is to happen when..